package.json

```diff
@@ -27,8 +27,9 @@
     "clean": "rm -rf dist/ && mkdir -p dist",
     "browserify:test": "browserify test/browser/test.js -t babelify --outfile dist/browser.test.js",
     "browserify:dist": "browserify src/client.js -t babelify --standalone feathers --outfile dist/feathers.js",
-    "browserify": "npm run browserify:dist && npm run browserify:test",
-    "add-dist": "npm run clean && npm run browserify:dist && git add dist/ --force && git commit -am \"Updating dist\"",
+    "browserify:min": "browserify src/client.js -t babelify --standalone dist/ | uglifyjs > dist/feathers.min.js",
```

Comment:
I just tried 1.4 and it seems to be working fine although the `--standalone` flag probably really isn't needed.